.github/workflows/java_binding_publish.yml

```diff
@@ -0,0 +1,140 @@
+name: Create Java release
+
+on:
+  workflow_dispatch:
+    paths:
+      - .github/workflows/java_binding_publish.yml
+      - bindings/java/**
+      - src/**
+      - Cargo.lock
+      - Cargo.toml
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  create-releases:
+    name: Create Java release
+    if: ${{ ! github.event.schedule }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest, macos-11]
```

Comment:
Why macos-latest and macos-11?

Reply:
For macOS we build 2 JARs:
- default target
- aarch64-apple-darwin target

Building both within the macos-latest run works, however the `svenstaro/upload-release-action@v2` can't handle  2 uploads within 1 run. So the only way I got it working was to use 2 separte OS runs

Reply:
Pls add a comment so it's also clear for other reading it

Reply:
added

Reply:
> the svenstaro/upload-release-action@v2 can't handle 2 uploads within 1 run

[softprops/action-gh-release](https://github.com/softprops/action-gh-release#%EF%B8%8F-uploading-release-assets) allows glob expressions or lists of files if that's what you're looking for. That would prevent the need to run two macOS instances in parallel as we have an organization-wide concurrency limit

Reply:
Thanks for the suggestion, using this now - works much better!